UpdateDB/DataLoading/AffixDataLoader.cs

```diff
@@ -233,13 +235,42 @@ private static IEnumerable<XmlStat> ExtractStats(string statColumn, string affix
                 }
                 stat = rangeRenameFunc(affix, stat);
                 var fromTo = stat.Split(new[] {" to "}, StringSplitOptions.None);
-                yield return new XmlStat
+                var from = fromTo[0].ParseFloat();
+                var to = fromTo.Length > 1 ? fromTo[1].ParseFloat() : from;
+                xmlStats.Add(new XmlStat
                 {
                     Name = affix,
-                    From = fromTo[0].ParseFloat(),
-                    To = fromTo.Length > 1 ? fromTo[1].ParseFloat() : fromTo[0].ParseFloat()
-                };
+                    From = new[] { from },
+                    To = new[] { to }
+                });
+            }
+
+            // merge "... # minimum ..., ... # maximum" into "... # to # ..."
+            if (xmlStats.Count > 1)
+            {
+                var previous = xmlStats[0];
+                for (var i = 1; i < xmlStats.Count; i++)
```

Comment:
So I just looked this up and apparently ReSharper does this kind of thing. I certainly don't want to tell you how to write your code, but I'd just like to point out that I found this method (the first major one I'm looking at) very hard to grasp, mainly because of the `var` spam. I don't mean that I don't understand a for loop (or that the code overall is hard to understand), but it's kind of like taking the colour out of a photo - of course you can still see what's going on, but it's certainly less enjoyable for others to look at.

Again, I'm not the one writing 10k LOC (and I realize half this function was already like this). I don't want to suggest you turn off that feature, just leaving the feedback here that it shifts the burden of "readable code" from the writer to the reader (in my eyes). Maybe the variable names could be more descriptive, but maybe that would distract more from what's going on. I don't know.